gems/aws-sdk-s3/CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 Unreleased Changes
 ------------------
 
+* Feature - Support S3 Access Grants authentication. Access Grants can be enabled with the `s3_access_grants` option, and custom options can be passed into the `access_grants_credentials_provider` option. This feature is enabled if `aws-sdk-s3control` is installed.
```

Comment:
The wording is a little confusing - s3_access_grants is always enabled if s3control is installed?  Should it be something like "this feature requires s3control to be installed"?